### ISSUE TITLE
Playlist not working on Android

### ISSUE THREAD

**Issue by noman5412** [RenderHeads/UnityPlugin-AVProVideo#2057]
PlaylistDemoScene is working perfectly on Editor but showing weird errors on Android I have tried with both mediapaths (URL paths and StreamingAssets) but facing same issue.
I have attached logcat so you can check.

 - Unity version: 2021.3.5f1
 - AVPro Video version (v3.1.1 (trial)): 
 - Operating system version: Android 
 - Device model: Android 12
 - Video specs (FullHD (1920x1080), 30fps, mp4): 

[
[com.Scientia.AVPro-logcat.txt](https://github.com/user-attachments/files/17597093/com.Scientia.AVPro-logcat.txt)
]

**Comment by MorrisRH** [comment]
Unable to reproduce this here using:
- Unity 2021.3.45f1
- AVPro Video 3.1.1 Ultra
- Vulkan and OpenGLES 3 renderers
- Demo_Playlist scene

I did spot in the log you provided that you're only using version 3.0.11 of the plugin:
```
AVProVideo] Initialising AVPro Video v3.0.11 (native plugin v3.0.11-trial) on Adreno (TM) 610/OpenGL ES 3.2 V@0615.81 (GIT@75362bd531, I5cf4e5bf0f, 1704968105) (Date:01/11/24) (MT True) on Android
```

**Comment by Ste-RH** [comment]
We have just released v3.1.2 @noman5412.

We would recommend you update to this version and re-test for the issue.
